src/metadata/ColumnMetadata.ts

```diff
@@ -592,7 +592,11 @@ export class ColumnMetadata {
                 value = this.referencedColumn.getEntityValue(PromiseUtils.extractValue(entity[this.propertyName]));
 
             } else {
-                value = entity[this.propertyName];
+                if (Buffer.isBuffer(entity)) {
+                    value = entity;
+                } else {
+                    value = entity[this.propertyName];
+                }
```

Comment:
I tried to change logic to not destructive.
What do you think?

Reply:
looks like a hack. I'll left this PR opened and take a look on a problem by myself once I get time for it.

Reply:
I got it 😃